### PR TITLE
created a ViewProfile, with readOnly input types.

### PR DESCRIPTION
**Pull request by mthembbu**
Additional info about the application status has been catered for as well
Bootstrap 4.3.1 has been included.

### CODE REVIEW

File: webapp/src/pages/viewprofile/components/ViewProfileComponent.js [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+        super(props);
+        this.state = {
+            user: {},
+            loading: true,
+            error:"",
+            isNull:true
+        }
+    }
+
+isUserNull(user){
```
Comment (avishkar58) [inline webapp/src/pages/viewprofile/components/ViewProfileComponent.js]: This is nit, but is this method necessary? I think it's clearer to just "inline" this and have have `result.data === null`  in your setState function directly.

File: webapp/src/pages/profileList/components/ProfileListComponent.js [hunk 2]
```diff
@@ -33,15 +33,20 @@ class ProfileListComponent extends Component{
                             });
                     });                
  }
-
+onsub=(user_id)=>{
```
Comment (avishkar58) [inline webapp/src/pages/profileList/components/ProfileListComponent.js]: Can this method have a more descriptive name please?

File: webapp/src/pages/viewprofile/components/ViewProfileComponent.js (hunk 1 above)
Comment (mthembbu) [reply webapp/src/pages/viewprofile/components/ViewProfileComponent.js]: dano....

File: webapp/src/pages/profileList/components/ProfileListComponent.js (hunk 2 above)
Comment (mthembbu) [reply webapp/src/pages/profileList/components/ProfileListComponent.js]: sorted

File: webapp/src/pages/profileList/components/ProfileListComponent.js [hunk 3]
```diff
@@ -33,15 +33,20 @@ class ProfileListComponent extends Component{
                             });
                     });                
  }
-
+onSubmit=(user_id)=>{
+    console.log(user_id)
```
Comment (jaderabbit) [inline webapp/src/pages/profileList/components/ProfileListComponent.js]: clean plz

File: webapp/src/services/profilelist/profilelist.service.js [hunk 4]
```diff
@@ -20,4 +21,18 @@ function getProfilesList(event_id){
                       ? error.response.data.message :
                          error.message }
         });
+}
+
+function getUserProfile(user_id){
+    return axios
+            .get(baseUrl+"/api/v1/userprofile?user_id="+user_id.toString().split(":")[1],{headers: authHeader()})
```
Comment (jaderabbit) [inline webapp/src/services/profilelist/profilelist.service.js]: @mthembbu Can you explain your reasoning here? Why do we need to make it a string and split it?

Comment (mthembbu) [reply webapp/src/services/profilelist/profilelist.service.js]: my mistake was not creating a new variable as an int, then parse that int instead.

**Pull request merged**